include/Customer.h

```diff
@@ -25,5 +47,9 @@ void from_json(const nlohmann::json & j, Customer & t) {
     j.at("id").get_to(t.id); 
     j.at("spent").get_to(t.spent); 
 }
+//** KOOS
+//** not sure if this is the right place dor a json convert? 
```

Comment:
This is how the nlohmann json works (it's been a while since I've used it) but all you really need to do is define a to_json and from_json when it comes to structs and it'll do the conversion under the hood.